.github/workflows/python-integration-tests.yml

```
@@ -37,7 +37,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install poetry pytest
-        cd python && poetry install --with hugging_face
+        cd python && poetry install --with hugging_face --with qdrant-client
```

Looks like the group is called 'qdrant' 

```suggestion
        cd python && poetry install --with hugging_face --with qdrant
```

---

I'll verify. Quick review in their demo and qdrant-client repo, it shows qdrant-client in poetry install. But will confirm, test the call above and change if applicable.

---

Okay there is no consistency noted, so I will change as you noted in upcoming commit.

---

Addressed and resolved in: https://github.com/microsoft/semantic-kernel/pull/919/commits/2d79e60727ea2dc9dd267bb35b298c069c32b5f3